apps/api/src/.env.test

```diff
@@ -1,6 +1,7 @@
 OOGLE_OAUTH_CLIENT_ID=11
 GOOGLE_OAUTH_CLIENT_SECRET=11
 GOOGLE_OAUTH_REDIRECT=http://localhost:3000/v1/auth/google/callback
+STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
```

Comment:
@djabarovgeorge could you also add this key to the validation we have in the `config` folder on the API? So that the server will throw an error on the bootstrap phase of this missing env variable.